solutions/USACO dec18-bucketlist.mdx

```diff
@@ -41,3 +41,47 @@ file_out.close()
 </PySection>
 
 </LanguageSection>
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <algorithm>
+using namespace std;
+int event[1001]; // declaring the array globaly to intialize all the elements to 0
+int main() {
+	freopen("blist.in", "r", stdin);
+	freopen("blist.out", "w", stdout); 
+	int maxtime = 1000;
+	int n; cin >> n;
+		
+		
+	for (int i = 0; i < n; i++) { // iterating through n lines to input the data in
+		int start; cin >> start; 
+		int end; cin >> end;
+		int amt; cin >> amt;
+		event[start] += amt; 
+		// simulating the events by adding/subtracting the amount from si and ti
+		event[end] -= amt; // ^
```

Comment:
what are si and ti?

Reply:
from the problem its what they call the input

Reply:
aren't si and ti the `start` and `end` in your code? you're not adding/subtracting from `start` and `end`...